ceci/stage.py

```diff
@@ -18,33 +18,42 @@ class PipelineStage:
     """
     parallel = True
     config_options = {}
+    doc=""
```

Comment:
Is there a reason not to use the docstring of the class?

Reply:
of course, that's just me being dumb